### PR TITLE
Add consist list for clusterclaims

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>
https://github.com/stolostron/backlog/issues/23826

### CODE REVIEW

**Comment by xuezhaojun** [comment]
Should we use `consistList` or `createOnlyList`?

**Comment by xuezhaojun** [comment]
We should require users use lable to do policy not using clusterclaim in 2.6.

File: pkg/klusterlet/clusterclaim/clusterclaimer.go [hunk 1]
```diff
@@ -51,6 +51,11 @@ const (
 	labelExcludeBackup = "velero.io/exclude-from-backup"
 )
 
+// consistList returns a list of cluster claims that only need to create, not update.
+var consistList = []string{
+	ClaimK8sID,
```
Comment (qiujian16) [inline pkg/klusterlet/clusterclaim/clusterclaimer.go:56]: what does consistList mean? clusterClaimCreateOnlyList?

Comment (xuezhaojun) [reply pkg/klusterlet/clusterclaim/clusterclaimer.go:56]: Yes, clusterClaimCreateOnlyList is better.

File: test/e2e/clusterclaim_test.go [hunk 2]
```diff
@@ -91,6 +91,43 @@ var _ = ginkgo.Describe("Testing ClusterClaim", func() {
 		}, eventuallyTimeout, eventuallyInterval).ShouldNot(gomega.HaveOccurred())
 	})
 
+	ginkgo.It("should not rollback the change of the ClusterClaim, if it's in consist list", func() {
```
Comment (zhujian7) [inline test/e2e/clusterclaim_test.go:94]: nit, CreateOnlyList ?

Comment (xuezhaojun) [reply test/e2e/clusterclaim_test.go:94]: Got, it I will fix it in the following up PR.

**Comment by zhujian7** [comment]
@xuezhaojun could you squash the commits?

File: test/e2e/clusterclaim_test.go (hunk 2 above)
Comment (xuezhaojun) [reply test/e2e/clusterclaim_test.go]: Fixed

**Comment by xuezhaojun** [comment]
> @xuezhaojun could you squash the commits?

Sure, fixed.

**Pull request merged**